internal/pkg/cli/project_delete.go

```diff
@@ -23,8 +23,8 @@ import (
 const (
 	defaultProfile = "default"
 
-	fmtConfirmProjectDeletePrompt = "Are you sure you want to delete project %s?"
-	confirmProjectDeleteHelp      = "Deleting a project will remove all associated resources. (apps, envs, etc.)"
+	fmtConfirmProjectDeletePrompt = "Are you sure you want to delete project %s? This will delete your project as well as any apps, environments, and pipelines."
+	confirmProjectDeleteHelp      = "Deleting a project will remove all associated resources. (apps, envs, pipelines, etc.)"
```

Comment:
Should we ditch this help message now that the main question is more illuminating?

Reply:
Hmmm, maybe? I don't see the harm in keeping it in there for consistency tho